### PR TITLE
List component

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description

[Create list component](https://trello.com/c/3WKW3Wce/186-186-create-list-component)

Create a list component that we can dynamically reference according to the data in the CMS.
image

## Acceptance Criteria

Should be able to pass in CSS classes in order to change sizing i.e. full width half width etc 

## Test Instructions

1. Test the component on different screen sizes
2. Test the component width, try changing the width

## Help Requested

After 4 items, the circles just stay dark blue. I'm not sure if we need to change this or not, any help appreciated!

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [x] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: components/molecules/List.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ *  List component
+ */
+export function List(props) {
+  let opacity = 20;
+  return (
+    <ol className={props.className}>
+      {props.items.map((item, key) => {
+        opacity += 20;
```
Comment (Moro-Code) [inline components/molecules/List.js:11]: Opacity is gonna start at 40 this way. Assign a variable to store the JSX and then add 20 to the opacity after. Note if the opacity === 100 .... 20 should not be added

**Review by Moro-Code — CHANGES_REQUESTED** [review]
Wow @EphraimAssi super great work! Code is very clean I'm impressed ! Just one thing... the opacity doesn't seem to be working and I left a very minor comment

File: components/molecules/List.js (hunk 1 above)
Comment (EphraimAssi) [reply components/molecules/List.js]: The opacity in Figma design starts at 40 that's why I left it that way. Is it still better to put the JSX in a variable?

**Review by Moro-Code — CHANGES_REQUESTED** [review]
Hey @EphraimAssi just need to fix the colours for the list item

What we want

Screen Shot 2021-05-20 at 9 11 21 AM

What we have

Screen Shot 2021-05-20 at 9 12 20 AM

**Review by will0684 — APPROVED** [review]
Looks good Ephraim nice work 👍

**Review by Moro-Code — APPROVED** [review]
LGTM!

**Pull request merged**